google/cloud/aiplatform/base.py

```diff
@@ -108,3 +292,231 @@ def resource_name(self) -> str:
     def display_name(self) -> str:
         """Display name of this resource."""
         return self._gca_resource.display_name
+
+
+class AiPlatformResourceNounWithFutureManager(AiPlatformResourceNoun, FutureManager):
```

Comment:
Will any resource still be under AiPlatformResourceNoun (without FutureManager)?  Such a resource would not have some public method such as `wait`.  Do the users have an easy way to tell if something is with/without future manager?

Reply:
Every resource noun should be with FutureManager.